js/TUI.js

```diff
@@ -12,28 +12,29 @@
  * 
  * TUI stands for Tweak UI.
  *
- * Requires js/util.js and the YUI Dom and Cookie libraries.
+ * Requires js/util.js and jquery
  *
  * See template/en/default/bug/create/create.html.tmpl for a usage example.
  */
 
 var TUI_HIDDEN_CLASS = 'bz_tui_hidden';
 var TUI_COOKIE_NAME  = 'TUI';
+var TUI_alternates   = {};
```

Comment:
This TUI_alternates thing is a pretty poor mechanism; the two sets of wordings should both be in the HTML in separate spans, and the addition or removal of a class in the parent element should switch the visibility of the two. But I guess you don't need to fix that in this change.